vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -208,6 +219,18 @@ def process_weights_after_loading(self, layer: Module) -> None:
             else:
                 layer.input_scale = None
 
+            # If rocm, adjust the scaling factor.
+            # For the same bits representation, e4m3fnuz value is half of
+            # the e4m3fn value, so we should double the scaling factor to
+            # get the same dequantized value.
+            # https://onnx.ai/onnx/technical/float8.html
+            if is_hip():
```

Comment:
Can we add a utility function in `w8a8_utils.py` called `convert_to_e4m3fnuz` for this?